nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md

```diff
@@ -16,7 +16,17 @@ Messages matching the following criteria will be auto subscribed at startup.
 
  1. Defined as an event either using `IEvent` or by the `.DefiningEventsAs` convention.
  1. At least one [message handler and/or saga](/nservicebus/handlers/) exists for the given event.
- 1. If the selected transport [does not support publish-subscribe natively](/transports/#types-of-transports-unicast-only-transports), the publisher for that message needs to be specified via the [routing](/nservicebus/messaging/routing.md) API.
+
+If a message handler is defined for an event but no publishers information can be found the endpoint will log an error, as the following one, at startup:
```

Comment:
I think this section needs to be changed a bit. The only time this warning is relevant is for message-based pubsub, so this mention of the warning should come after the discussion about needing to use `RegisterPublisher`. Then, you can point out that if `RegisterPublisher` hasn't been called, the warning will be logged.

Reply:
Yeah, better.